### DISCUSSION TITLE
Please Read Makers

### DISCUSSION THREAD

**Discussion by Rainzy54** [3kh0/ext-remover#66] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am just wondering if you are making a new unblocker or you have just stopped making new codes, im just wondering.

**Comment by 3kh0** [comment, marked as answer]
Join the tn discord for that [discord.gg/unblock](https://discord.gg/unblock)

**Comment by Aka-but-nice** [comment]
no
